src/status_im/ui/screens/wallet/components/styles.cljs

```diff
@@ -107,11 +107,13 @@
 
 (def wallet-value
   {:padding-left   6
-   :flex-shrink    1
    :color          styles/color-white-transparent-5
    :font-size      15
    :letter-spacing -0.2})
 
+(def wallet-value-amount
+  {:flex -1})
```

Comment:
could you elaborate?

Reply:
This allows text to shrink properly when too long.

Reply:
just wondering why is it better than `:flex-shrink    1`,

Reply:
It's better because it works :)
That's the only way I could have the amount shrink but keep `ETH` intact and visible.

Reply:
that's weird, i tested it and it worked

Reply:
I changed the behaviour so that it is `1.11111111.... ETH`